assets/components/organisms/breadcrumbs/breadcrumbs.scss

```diff
@@ -88,3 +90,47 @@
     color: $gray-600;
   }
 }
+
+// create space out of container on the left for hamburger button
+.breadcrumb-container nav {
+  position: relative;
+  margin-left: -1 * ($hamburger-btn-size + (0.5 * $spacer));
+  padding-left: $hamburger-btn-size + (0.6 * $spacer);
+  margin-top: 0.5 * $spacer;
+}
+
+// hide hamburger when in nav solid context
+.nav-solid #hamburger {
+  display: none;
+}
+
+// hamburger button
+#hamburger {
```

Comment:
![OMG](https://media.giphy.com/media/fpXxIjftmkk9y/giphy.gif)

Reply:
Be more specific? naming? use of ids? 
(the use of ID is because it will probably be good for it to be accessible through the hypothetic skiplinks)